<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-23T17:46:51 IST" name="Default suite" finished-at="2022-08-23T17:47:49 IST" duration-ms="57963">
    <groups>
    </groups>
    <test started-at="2022-08-23T17:46:51 IST" name="Default test" finished-at="2022-08-23T17:47:49 IST" duration-ms="57963">
      <class name="com.tcw.testcases.ScheduleTest">
        <test-method is-config="true" signature="settingTheLogs()[pri:0, instance:com.tcw.testcases.ScheduleTest@77128536]" started-at="2022-08-23T17:46:50 IST" name="settingTheLogs" finished-at="2022-08-23T17:46:51 IST" duration-ms="58" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- settingTheLogs -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.tcw.testcases.ScheduleTest@77128536]" started-at="2022-08-23T17:46:51 IST" name="setUp" finished-at="2022-08-23T17:46:57 IST" duration-ms="6714" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addScheduleForEmpTest(java.util.Hashtable)[pri:2, instance:com.tcw.testcases.ScheduleTest@77128536]" started-at="2022-08-23T17:46:57 IST" name="addScheduleForEmpTest" data-provider="dp2" finished-at="2022-08-23T17:47:48 IST" duration-ms="50602" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{customPay=52.36, desiredDate=2022-08-13, PesonalNote=Your Notes, Notes=My Notes, jobDrpDwn=Home, desiredEmp=Urvashi Jk, endTime=05:30 PM, stTime=03:00 PM, locDrpDwn=India}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [SCHEDULE OVERLAP !! !!] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [SCHEDULE OVERLAP !! !!] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.tcw.testcases.ScheduleTest.addScheduleForEmpTest(ScheduleTest.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addScheduleForEmpTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.tcw.testcases.ScheduleTest@77128536]" started-at="2022-08-23T17:47:48 IST" name="tearDown" finished-at="2022-08-23T17:47:49 IST" duration-ms="474" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcw.testcases.ScheduleTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
